.gitignore

```diff
@@ -36,3 +36,6 @@ yarn-error.log*
 
 # editor formatting
 .prettierrc.js
+
+# graphql-codegen
+generated
```

Comment:
Want to .gitignore the output. This is why the `generate` command is both `predev`, and `prebuild`